### PR TITLE
[WIP] Start working on making metrics use AJAX and group by hour, day, month

### PR DESCRIPTION
**Pull request by jbrooksuk**
Completely a WIP at the moment.

image

### CODE REVIEW

File: resources/views/partials/metrics.blade.php [hunk 1]
```diff
@@ -19,56 +12,98 @@
                 </h4>
             </div>
             <div class="col-xs-2 text-right">
-                <small>{{ trans('cachet.metrics.filter.hourly') }}</small>
+                <div class="dropdown">
+                    <a href="javascript: void(0);" class="btn btn-default dropdown-toggle" data-toggle="dropdown">{{ trans('cachet.metrics.filter.hourly') }} <span class="caret"></span></a>
```
Comment (GrahamCampbell) [inline resources/views/partials/metrics.blade.php]: what is this?

Comment (GrahamCampbell) [inline resources/views/partials/metrics.blade.php]: lol

Comment (jbrooksuk) [inline resources/views/partials/metrics.blade.php]: Lol unfinished.

**Comment by jbrooksuk** [comment]
Don't close this one :)

**Comment by GrahamCampbell** [comment]
I'm not. I was closing and opening.

**Pull request closed without merging**